ui/tests/integration/components/config-ui/messages/page/create-and-edit-message-test.js

```diff
@@ -112,6 +118,8 @@ module('Integration | Component | messages/page/create-and-edit-message', functi
       message: 'Blah blah blah. Some super long message.',
       start_time: '2023-12-12T08:00:00.000Z',
       end_time: '2023-12-21T08:00:00.000Z',
+      link_title: 'Learn more',
```

Comment:
I got hit with this in HDS components, but I think we're moving away from Learn more language to be more specific. I got hit specifically for an External Link on replication. But maybe we should clarify when we should or should not use Learn more. [Here](https://github.com/hashicorp/vault/pull/24230/files#r1402696830) is the my PR with the comment.